### PR TITLE
Exposing More Data Field in 802.11 Header

### PR DESCRIPTION
**Pull request by juno0812**
Added getter, setter, and tests for the more data field of the 802.11 header.

### CODE REVIEW

File: src/dot11/dot11_base.cpp [hunk 1]
```diff
@@ -164,6 +164,10 @@ void Dot11::power_mgmt(small_uint<1> new_value) {
     header_.control.power_mgmt = (new_value)? 1 : 0;
 }
 
+void Dot11::more_data(small_uint<1> new_value) {
+    header_.control.more_data = (new_value)? 1 : 0;
```
Comment (mfontanini) [inline src/dot11/dot11_base.cpp]: You should be able to just do `= new_value` as that is already in the range [0, 1].

**Comment by mfontanini** [comment]
Thanks, I'm not sure how this one was missed.

File: src/dot11/dot11_base.cpp (hunk 1 above)
Comment (juno0812) [reply src/dot11/dot11_base.cpp]: I agree.  I was just keeping it consistent with the other fields like power management above.

Comment (mfontanini) [reply src/dot11/dot11_base.cpp]: Oh I didn't realize that was the case. Alright, I'll merge it as it is then.

**Pull request merged**